.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
```

Comment:
I think you're a little confused about how that rule works. Each app/addon needs to add its own name to the list of local prefixes in order for it to work properly. That's all that this file is doing, changing the settings for this addon, right? It doesn't change it for anyone else. There's no way for `eslint-config-frost-standard` to include the name of the consuming addon/app in it's own `eslint.json`, so consumers need to specify it themselves. If we used a `.js` file instead of a `.json` file in `eslint-config-frost-standard`, we might be able to do something automatically, but as it stands now, consumers are encouraged to make the customizations seen here.